eth/core/state/journal/transient_storage_test.go

```diff
@@ -46,41 +46,41 @@ var _ = Describe("TransientStorage", func() {
 	It("should add without impacting previous state", func() {
 		ts.SetTransientState(alice, key, value)
 		ts.SetTransientState(bob, key, value)
-		Expect(ts.PeekAt(0).Get(bob, key), common.Hash{})
+		Expect(ts.PeekAt(0).Get(bob, key)).To(Equal(common.Hash{}))
 	})
 
 	It("should have consistent gets and sets", func() {
 		ts.SetTransientState(alice, key, value) // {alice:value}
-		Expect(ts.GetTransientState(alice, key), value)
+		Expect(ts.GetTransientState(alice, key)).To(Equal(value))
 
 		before := ts.Snapshot()
 		ts.SetTransientState(alice, key, value2) // {alice:value2}
-		Expect(ts.GetTransientState(alice, key), value2)
+		Expect(ts.GetTransientState(alice, key)).To(Equal(value2))
 
 		ts.SetTransientState(bob, key, value) // {alice:value2, bob: value}
 		ts.RevertToSnapshot(before)           // {alice:value}
-		Expect(ts.GetTransientState(alice, key), value)
-		Expect(ts.GetTransientState(bob, key), common.Hash{})
+		Expect(ts.GetTransientState(alice, key)).To(Equal(value))
+		Expect(ts.GetTransientState(bob, key)).To(Equal(common.Hash{}))
 	})
 
 	It("should correctly finalize", func() {
 		ts.SetTransientState(alice, key, value)
 		ts.Finalize()
-		Expect(ts.Size(), 0)
+		Expect(ts.Size()).To(Equal(0))
 		Expect(func() { ts.Finalize() }).ToNot(Panic())
 	})
 
 	It("should correctly clone", func() {
 		ts.SetTransientState(bob, key, value)
-		Expect(ts.PeekAt(0).Get(bob, key), common.Hash{})
-		Expect(ts.GetTransientState(bob, key), value)
```

Comment:
@calbera I think you had a bug in your test that was being hidden by the incorrect `Expect` usage. Please verify the correctness here.

Reply:
yeah this is the correct replacement, thanks @mmsqe for the fix!